requirements.txt

```diff
@@ -9,5 +9,5 @@ exchangelib
 django-markdownx
 django-crispy-forms
 psycopg2-binary
-git+https://github.com/openego/oedialect
-git+https://github.com/henhuy/python-highcharts.git
\ No newline at end of file
+git+https://github.com/openego/oedialect#egg=oedialect
```

Comment:
this could not be replaced simply by oedialect? It was [updated](https://pypi.org/project/oedialect/) recently

Reply:
maybe - I did not know about that. We should check.

Reply:
@Bachibouzouk, the `psycopg2` requirement is currently updated to `psycopg2-binary` in oedialect (cf. https://github.com/OpenEnergyPlatform/oedialect/pull/25) but is not part of the curent v0.0.6dev, right? So we'll have to wait for the merge and successive release to avoid install conflicts with the WAM (which already uses recent `psycopg2-binary`).

Reply:
@Bachibouzouk @MGlauer Do you know when we can expect a new oedialect release? As stated in #88 I'd like to release WAM v0.1.3 pretty soon.. :)

Reply:
> One more thing @henhuy You removed python-highcharts but even with external loading of HC (#78) this package is needed for the mapping, right?

You are right - package must still be there. sorry

Reply:
> You are right - package must still be there. sorry

Although it's not shown in the changes above, it's still [there](https://github.com/rl-institut/WAM/blob/features/packaging/requirements.txt)... strange
my bad

Reply:
> @Bachibouzouk @MGlauer Do you know when we can expect a new oedialect release? As stated in #88 I'd like to release WAM v0.1.3 pretty soon.. :)

Oh, I overlooked that it was changed in v0.0.6dev in the setup.py.